.gitignore

```diff
@@ -6,3 +6,4 @@ __pycache__/
 .idea
 env
 dist
+.DS_Store
```

Comment:
You should probably add `.DS_Store` to your global `.gitignore` file, so you don't always have to ignore it for every repo.

http://devoh.com/blog/2013/01/global-gitignore